examples/mcp/src/server.ts

```diff
@@ -9,7 +9,7 @@ type Env = {
 type State = { counter: number };
 
 export class MyMCP extends McpAgent<Env, State, {}> {
-  server = new McpServer({
+  mcp = new McpServer({
```

Comment:
Maybe mcp could be a getter that returns server? That way it would be totally backward compatible.

Reply:
I don't think that would allow me to do this:

```ts
export class MyMCP extends McpAgent {
	mcp = new McpServer({}); // <-- this is the desired API

	async init() {
		this.mcp.server.setRequestHandler(
			SetLevelRequestSchema,
			async (request, extra) => {
				// ... handler logic
			},
		);
	}
}
```

Reply:
I could do that myself without having to change anything in `agents`. That's actually a reasonable workaround if they decide this is more trouble than it's worth. Thanks for the idea!

Reply:
Oh, actually I can't do that because `mcp` is already set to the `mcpClientManager` 🙁